packages/core/src/tools/error.spec.ts

```diff
@@ -33,7 +42,13 @@ describe('formatUnknownError', () => {
       ],
     }
 
-    const formatted = formatUnknownError(stack, undefined, 'Uncaught')
+    const formatted = computeRawError({
+      stackTrace: stack,
+      error: undefined,
+      startClocks: clocksNow(),
+      nonErrorPrefix: 'Uncaught',
+      source: ErrorSource.CUSTOM,
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏In order to avoid to clutter the tests with non relevant parameters, you could extract some default raw errors params.
It would make clearer for each case what are the important inputs of the test subject.
```suggestion
    const formatted = computeRawError({
      ...DEFAULT_RAW_ERROR_PARMS,
      stackTrace: stack,
      error: undefined,
    })
```